xrmocap/utils/time_utils.py

```diff
@@ -0,0 +1,91 @@
+import time
+from xrprimer.utils.log_utils import get_logger, logging
+
+
+class Timer:
+
+    def __init__(self, name: str, logger: logging.Logger = None) -> None:
```

Comment:
logger's type could be Logger, str(logger name) or None. Thus, we could revise typing hint and docstring according to https://github.com/openxrlab/xrmocap/blob/main/xrmocap/ops/projection/aniposelib_projector.py#L27